test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -57,7 +57,20 @@ class test_ucp_tag_offload : public test_ucp_tag {
 
     void activate_offload(entity &se, ucp_tag_t tag = 0x11)
     {
-        send_recv(se, tag, receiver().worker()->context->config.ext.tm_thresh);
+        size_t tm_thresh = receiver().worker()->context->config.ext.tm_thresh;
+
+        // Send to ensure that wireup's UCT iface has been closed and
+        // it is not considered for num_active_iface on worker
+        // (message has to be less than `UCX_TM_THRESH` value)
+        send_recv(se, tag, ucs_min(0ul, tm_thresh - 1));
+
+        if (receiver().worker()->num_active_ifaces > 1) {
```

Comment:
why is current PR with memtype TLS exposing the need to adjust this test?

Reply:
due to `rocm_gdr` is not disabled now (and we have 2 active ifaces). Already disabled this in ucp_test.cc, but still making this test more robust..